.gitignore

```diff
@@ -9,6 +9,7 @@
 bundle/*
 vendor/bundle
 
+.env.local
 # Ignore environment files (except test for CI)
 ,env
 .env.development
```

Comment:
How about this instead: of lines 12 onward.  I think the comma in line 14 is a bug anyway.

```
# Ignore environment files (except test for CI)
.env*
!.env.test
```